src/commands.rs

```diff
@@ -66,7 +66,20 @@ pub fn run(command: Command, flags: &oof::Flags) -> crate::Result<()> {
 
                 // Breaks if Lzma is .lz or .lzma and not .xz
                 // Or if Bzip is .bz2 and not .bz
```

Comment:
Hey, am back, I recently took a break from `ouch`.

So apparently #96 was fixed by changes made by @SpyrosRoum at #133, if you check this comments above the lines you edited, it references that might panic if extension is written in an alternate way, like LZMA or BZIP because those have these other aliases, `.lz` `.lzma` and `.xz` are the same thing for `ouch`.